app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt

```diff
@@ -1,11 +1,15 @@
-package org.simple.clinic.monthlyReports.questionnaire.entry
+package org.simple.clinic.monthlyscreeningreports.form.entry
```

Comment:
If there's no `entry` namespace in the `src` folder, maybe remove it from test as well?

Reply:
@olttwa yeah, we can do that.

Reply:
@olttwa fixed